_posts/2019-11-11-issue-100.md

```diff
@@ -40,4 +40,4 @@ If you like hand-coding WebAssembly properly, by which I mean properly, not in W
 
 ### And Finally
 
-Almost as exciting as this newsletter hitting issue #100 is the news that [in-browser debugging of WebAssembly is starting to ship](https://twitter.com/WasmWeekly/status/1193090880523186176).
\ No newline at end of file
+Almost as exciting as this newsletter hitting issue #100 is the news that [in-browser debugging of WebAssembly is starting to ship](https://twitter.com/WasmWeekly/status/1193090880523186176).
```

Comment:
Sorry, I'm not sure what this change is about.
Maybe just manually fix the Wasm3 chapter ;)